core/iwasm/interpreter/wasm_interp_fast.c

```diff
@@ -3840,7 +3879,7 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
                 HANDLE_OP_END();
             }
 
-            HANDLE_OP(WASM_OP_SET_LOCAL)
+            HANDLE_OP(WASM_OP_SET_LOCAL) { opcode = WASM_OP_SET_LOCAL; }
             HANDLE_OP(WASM_OP_TEE_LOCAL)
```

Comment:
Had better:
```C
            HANDLE_OP(WASM_OP_SET_LOCAL)
            {
                 opcode = WASM_OP_SET_LOCAL;
                 goto handle_op_set_tee_local;
            }
            HANDLE_OP(WASM_OP_TEE_LOCAL)
            {
                opcode = WASM_OP_TEE_LOCAL;
handle_op_set_tee_local:
}
```
Or the opcode may be always WASM_OP_SET_LOCAL after it is set?

Reply:
Done, thanks